### PR TITLE
Api testing

### PR DESCRIPTION
**Pull request by grus4**
"Added 'Adding a new Car and intercept the request' and API tests"

### CODE REVIEW

File: cypress/e2e/tests/apiTests.cy.js [hunk 1]
```diff
@@ -0,0 +1,129 @@
+describe('API testing', () => {
+    const baseUrl = 'https://jsonplaceholder.typicode.com/posts';
+    it('getting a list of post', () => {
+        cy.api({
+            url: `${baseUrl}`,
```
Comment (sttaran) [inline cypress/e2e/tests/apiTests.cy.js:5]: you don't need template string here, just pass `baseUrl`

File: cypress/e2e/tests/apiTests.cy.js [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+            method: 'GET',
+        }).as('listOfPosts');
+
+        cy.get('@listOfPosts').should(response => {
+            expect(response.status).to.eq(200);
+            expect(response.body).to.be.an('array');
+            expect(response.body.length).to.be.greaterThan(0);
```
Comment (sttaran) [inline cypress/e2e/tests/apiTests.cy.js:12]: also you can check if each object inside has particular keys

**Review by sttaran — APPROVED** [review]

File: cypress/e2e/tests/apiTests.cy.js (hunk 1 above)
Comment (grus4) [reply cypress/e2e/tests/apiTests.cy.js:5]: gotcha, appreciate it

File: cypress/e2e/tests/apiTests.cy.js (hunk 2 above)
Comment (grus4) [reply cypress/e2e/tests/apiTests.cy.js:12]: gotcha, appreciate it

**Pull request merged**